### ISSUE TITLE
Location icon moved over the map while dragging.

### ISSUE THREAD

**Issue by doozMen** [mapbox/mapbox-maps-ios#563]
https://user-images.githubusercontent.com/686038/127128745-1f3739bb-7015-4ccb-a8e2-727b8e1cfb73.mov

## Environment

- Xcode version:
- iOS version: 14.6
- Devices affected: iphone
- Maps SDK Version: 6.3.0

## Observed behavior and steps to reproduce

See video

## Expected behavior

Location should not move as you drag the map.

## Notes / preliminary analysis

I looked into the code and the users location is just a view above the map. It would seam better if the user location is rendered inside the map. Is this fixed with the beta?

**Comment by macdrevx** [comment]
Yes, this is fixed in v10. I'm going to transfer this issue to the pre-v10 repo.
